client/editor/map_editor.cpp

```diff
@@ -1,15 +1,16 @@
 // SPDX-License-Identifier: GPLv3-or-later
 // SPDX-FileCopyrightText: James Robertson <jwrober@gmail.com>
 
+// Editor
 #include "editor/map_editor.h"
 
-// client
-#include "citydlg.h"
-#include "client_main.h"
-#include "fc_client.h"
-#include "minimap_panel.h"
-#include "page_game.h"
-#include "views/view_map.h"
+// Client
+#include "citydlg.h"        //showEvent
+#include "client_main.h"    //check_open
+#include "fc_client.h"      //check_open
+#include "minimap_panel.h"  //showEvent
+#include "page_game.h"      //showEvent
+#include "views/view_map.h" //showEvent
```

Comment:
Not sure what these are for... I would interpret them as e.g. `minimap_panel::showEvent` being used, but this is not the case.

Reply:
This is to help remind me why I include the file. I've been hacking by adding varioud `.h` files to try different things so this helps me know which ones are vetted.